node_modules/oae-authentication/lib/api.js

```diff
@@ -677,6 +680,123 @@ var getUserIdFromLoginId = module.exports.getUserIdFromLoginId = function(tenant
     _getUserIdFromLoginId(loginId, callback);
 };
 
+
+/**
+ * Get the secret for an existing user's resetpassword request
+ *
+ * @param  {Context}    ctx             Standard context object containing the current user and the current tenant
+ * @param  {String}     tenantAlias     The tenant of the login id
+ * @param  {String}     userName        The user's own unique username 
+ * @param  {Function}   callback        Standard callback function
+ * @param  {Object}     callback.err    An error that occurred, if any
+ * @param  {Object}     callback.secret The secret token that will be returned
+ */
+var getResetPasswordSecret =module.exports.getResetPasswordSecret = function(ctx, tenantAlias, username, callback){
+	
+    // Associate the userID with loginID for later operations
```

Comment:
You can change this to `// Default the tenant alias to the current tenant's alias`